### PR TITLE
Add custom subsection in education cards

### PR DESCRIPTION
**Pull request by BernatBC**
Adds config for the following feature: hugo-toha/toha/pull/888

### CODE REVIEW

File: content/posts/configuration/sections/education/index.es.md [hunk 1]
```diff
@@ -56,6 +56,11 @@ degrees:
     url: "#"
   - title: Nullam vitae orci tincidunt purus viverra pulvinar.
     url: "#"
+  custonSections: #(opcional)
+    - name: Tesis
```
Comment (hossainemruz) [inline content/posts/configuration/sections/education/index.es.md:60]: Look like a typo here.

**Review by hossainemruz — COMMENTED** [review]
Thank you @BernatBC. Only a typo to fix. Otherwise, looks good.

File: content/posts/configuration/sections/education/index.es.md (hunk 1 above)
Comment (BernatBC) [reply content/posts/configuration/sections/education/index.es.md:60]: In spanish it's written like this. Thank you for checking it anyway!

Comment (hossainemruz) [reply content/posts/configuration/sections/education/index.es.md:60]: Ah. Thank you for explaining.

**Review by hossainemruz — APPROVED** [review]

**Pull request merged**